flink-connector-db2-cdc/src/main/java/com/ververica/cdc/connectors/db2/table/Db2TableSourceFactory.java

```diff
@@ -113,6 +115,18 @@ public DynamicTableSource createDynamicTableSource(Context context) {
         ResolvedSchema physicalSchema = context.getCatalogTable().getResolvedSchema();
         StartupOptions startupOptions = getStartupOptions(config);
 
+        OptionUtils.printOptions(
```

Comment:
The `config` from `helper.getOptions()` is actually an instance of `Configuration`. We could cast it to `Configuration` then get a map.